ui/app/components/app/permissions-connect-header/permissions-connect-header.component.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import IconWithFallBack from '../../ui/icon-with-fallback'
+
+export default class PermissionsConnectHeader extends Component {
+  static propTypes = {
+    icon: PropTypes.string,
+    iconName: PropTypes.string,
```

Comment:
This should be a required prop, as this should always be the origin (which we are guaranteed to have).

Reply:
done in a1d09c479